sportsref/ncaaf/teams.py

```diff
@@ -75,4 +75,28 @@ def sos(self, year):
             print "ERROR: NO SRS FOUND FOR {} in {}".format(self.teamID, year)
             return np.nan
 
-# TODO - BEGIN HERE
+    def get_roster(self, year):
+        """ Returns the roster for a team for the year
+
+        :param year: the year of the roster
+        :return: a dataframe with the player_id, class, and position
+        """
+        url = sportsref.ncaaf.BASE_URL +\
```

Comment:
I think using \ here is frowned upon in Python (that's at least my impression). I think using parentheses is preferred.